.github/CODEOWNERS

```diff
@@ -190,9 +190,9 @@
 
 
 # Global components and filters.
-/core/templates/components/button-directives/exploration-embed-button.service.ts @ankita240796
-/core/templates/components/button-directives/exploration-embed-button.directive.html @ankita240796
-/core/templates/components/button-directives/exploration-embed-button-modal.controller*.ts @ankita240796
+/core/templates/components/button-directives/exploration-embed-button-modal.component.html @ankita240796
+/core/templates/components/button-directives/exploration-embed-button-modal.component.spec.ts @nithusha21
+/core/templates/components/button-directives/exploration-embed-button-modal.component.ts @ankita240796
 /core/templates/components/button-directives/social-buttons.directive.ts @ankita240796
 /core/templates/components/button-directives/social-buttons.directive.html @ankita240796
```

Comment:
Remove the above 5 lines and assign `/core/templates/components/button-directives/` to ankita240796.

Reply:
Done in #9784